tests/sources/test_mimicry_source.py

```diff
@@ -19,3 +21,38 @@ async def test_fetching_data():
     val, _ = await source.fetch_new_datapoint()
     print(val)
     assert isinstance(val, float)
+
+
+@pytest.mark.skip("Requires API key")
```

Comment:
Maybe add to github ci via env var? Also, did you add api key to tellor shared doc?